BikeData/src/main/scala/by/itechart/Columns.scala

```diff
@@ -0,0 +1,21 @@
+package by.itechart
+
+object Columns extends Enumeration {
+  type TableCSV = Value
+
+  val columnIdZero = Value(0)
```

Comment:
It's good that you decided not to use indexes as literals in the code - such literals are called "magic numbers" and considered to be anti-patterns
But the variable with name columnIdFirst  is not better when just column(0).
What's inside first column? How could I use these values?

Name of variable should match semantics of data it stores. Is it recordIdColumnIndex or startTimeCololumnIndex?